src/ebpf.rs

```diff
@@ -195,8 +195,8 @@ pub const BPF_JSGT: u8 = 0x60;
 pub const BPF_JSGE: u8 = 0x70;
 /// BPF JMP operation code: syscall function call.
 pub const BPF_CALL: u8 = 0x80;
-/// BPF JMP operation code: return from program.
-pub const BPF_EXIT: u8 = 0x90;
+/// BPF JMP operation code: return from program (V1) or syscall (V2).
+pub const BPF_EXIT_SYSCALL: u8 = 0x90;
```

Comment:
I think its better to split it into two adjacent aliasing lines:
`pub const BPF_EXIT: u8 = 0x90;`
`pub const BPF_SYSCALL: u8 = 0x90;`

Reply:
Fixed.